Rakefile

```diff
@@ -121,31 +112,25 @@ namespace :test do
       'test/functional/**/*_test.rb',
       'lib/plugins/inspec-*/test/functional/**/*_test.rb',
     ])
-    t.warning = false # This just complains about things in underlying libraries
+    t.warning = true
```

Comment:
YESSSS. Thank you. That comment is so not true. While this is gonna be gross, it'll be worth it.